app/questionnaire/placeholder_transforms.py

```diff
@@ -243,3 +244,8 @@ def format_name(
     @staticmethod
     def _create_hyperlink(href: str, link_text: str) -> str:
         return f'<a href="{href}">{link_text}</a>'
+
+    @staticmethod
+    def list_item_count(list_to_count) -> int:
+        if list_to_count is not None:
+            return len(list_to_count) if list_to_count else 0
```

Comment:
This should always return an `int`. The first if is not needed, but this could be simplified with the need for `if`.
```suggestion
    def list_item_count(list_to_count: Optional[Sized]) -> int:
        return len(list_to_count or [])
```